Cargo.toml

```diff
@@ -15,7 +15,8 @@ resolver = "2"
 
 [workspace.dependencies]
 tlvc = { git = "https://github.com/oxidecomputer/tlvc.git", branch = "main" }
-hubtools = { git = "https://github.com/oxidecomputer/hubtools.git", branch = "main"}
+# TODO: Change to main branch before merge
+hubtools = { git = "https://github.com/oxidecomputer/hubtools.git", branch = "epoch"}
```

Comment:
I have that in there to make  sure I'm using the same version of hubtools across all the affected crates during development. Otherwise, it doesn't affect MGS.

Reply:
I can do without the hubtools bump.
We talked about just merging hubris (which implies MGS) at hubtools 0.4.6.
That change has been made.